app/src/main/java/com/mensinator/app/settings/SettingsViewModel.kt

```diff
@@ -210,6 +222,11 @@ class SettingsViewModel(
         return value
     }
 
+    private fun getString(settingKey: String): String {
+        val string = periodDatabaseHelper.getSettingByKey(settingKey)?.value.toString()
```

Comment:
Here a `String?` is converted to a `String` by calling `.toString()`. Not good practice, as there's no explicit fallback. Instead do something like `getSettingByKey(settingKey)?.value ?: "Unknown"` or `?: ""`

PS: Even better - when it is null, log the requested `settingKey` and that it is null.

Reply:
this should be covered by the new `getStringSettingByKey` func now :)